From: Punit Agrawal <punit@debian.org>
Date: Sun, 16 Feb 2020 19:39:33 +0900
Subject: pygments_parser: Convert to using python3

The pygments_parser.py script supports both python2 and
python3. Switch over to using python3 in preparation for removal of
python2 support in Debian.
---
 plugin-factory/pygments_parser.py.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/plugin-factory/pygments_parser.py.in b/plugin-factory/pygments_parser.py.in
index 36b29b8..3047ccd 100644
--- a/plugin-factory/pygments_parser.py.in
+++ b/plugin-factory/pygments_parser.py.in
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 #
 # Copyright (c) 2014
 #	Yoshitaro Makise
